tests/Rector/Class_/AddInterfaceToClassExtendingType/AddInterfaceToClassExtendingTypeTest.php

```diff
@@ -6,20 +6,19 @@
 
 use Iterator;
 use Rector\Testing\PHPUnit\AbstractRectorTestCase;
-use Symplify\SmartFileSystem\SmartFileInfo;
 
 final class AddInterfaceToClassExtendingTypeTest extends AbstractRectorTestCase
 {
     /**
      * @dataProvider provideData()
      */
-    public function test(SmartFileInfo $fileInfo): void
+    public function test(string $file): void
     {
-        $this->doTestFileInfo($fileInfo);
+        $this->doTestFile($file);
     }
 
     /**
-     * @return Iterator<SmartFileInfo>
+     * @return Iterator<string>
```

Comment:
Wondering with
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/40125720/203925926-7e29c9fd-3a6a-41e5-849f-70768ec620fd.png">
vs

[this](https://github.com/rectorphp/rector/blob/0.14.8/packages/Testing/PHPUnit/AbstractRectorTestCase.php#L81)